src/App.js

```diff
@@ -1,13 +1,129 @@
 import React from 'react';
+// import ReactDOM from 'react-dom';
+import ToDoNow from './components/TodoComponents/TodoList';
+import './components/TodoComponents/Todo.css';
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
+  constructor(props) {
+    super(props);
+    this.state = {
+      toDoList: [
+        {taskToDo: 'Component Code Fu',
+        id: 1323817077286,
+        completed: "false"
+      },
+        {taskToDo: 'Sand The Floor',
+        id: 1528817687286,
+        completed: "false"
+      },
+        {taskToDo: 'Paint The Fence',
+        id: 1428817077286,
+        completed: "false"
+      },  
+        {taskToDo: 'Wax On',
+        id: 1522818057286,
+        completed: "false"
+      },
+        {taskToDo: 'Wax Off',
+        id: 1598817077186,
+        completed: "false"
+      },
+      ],
+      newTask: '',
+      newId: Math.floor((Math.random() * 1000000000000) + 1000000000000),
+      newCompleted: "false",
+      newDeleted: "true",
+      textDecoration: 'line-through',
+    }
```

Comment:
only dynamic variables should be on `state` newCompleted, newDeleted, textDecoration won't change so they should be state variables